roles/pulibrary.solrcloud/defaults/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+solr_cloud_version: 7.1.0
+
+## Service options
+
+# Owner
+solr_user: solr
+solr_group: solr
+
+# start on boot
+solr_service_enabled: 'yes'
+# current state: started, stopped
+solr_service_state: started
+
+# Files & Paths
+solr_log_dir: /solr/logs
+solr_home: /solr
+solr_data_dir: '{{ solr_home }}/data'
+
+# Ports
+solr_port: 8983
+solr_jmx_enabled: 'true'
+solr_jmx_port: 1099
+solr_host: '{{ inventory_hostname }}'
+
+# JVM
+solr_gc_tune: "-XX:NewRatio=3 \
+-XX:SurvivorRatio=4 \
+-XX:TargetSurvivorRatio=90 \
+-XX:MaxTenuringThreshold=8 \
+-XX:+UseConcMarkSweepGC \
+-XX:+UseParNewGC \
+-XX:ConcGCThreads=4 -XX:ParallelGCThreads=4 \
+-XX:+CMSScavengeBeforeRemark \
+-XX:PretenureSizeThreshold=64m \
+-XX:+UseCMSInitiatingOccupancyOnly \
+-XX:CMSInitiatingOccupancyFraction=50 \
+-XX:CMSMaxAbortablePrecleanTime=6000 \
+-XX:+CMSParallelRemarkEnabled \
+-XX:+ParallelRefProcEnabled"
+# -XX:+AggressiveOpts \
+# -XX:+UseFastAccessorMethods \
+# -XX:+UseTLAB"
+solr_stack_size: 256k
+solr_heap: 512m
+solr_jetty_threads_min: 10
+solr_jetty_threads_max: 10000
+solr_jetty_threads_idle_timeout: 5000
+solr_jetty_threads_stop_timeout: 60000
+solr_jetty_secure_port: 8443
+solr_jetty_output_buffer_size: 32768
+solr_jetty_output_aggregation_size: 8192
+solr_jetty_request_header_size: 8192
+solr_jetty_response_header_size: 8192
+solr_jetty_send_server_version: 'false'
+solr_jetty_send_date_header: 'false'
+solr_jetty_header_cache_size: 512
+solr_jetty_delay_dispatch_until_content: 'false'
+solr_jetty_http_selectors: -1
+solr_jetty_http_acceptors: -1
+
+# ZOOKEEPER
+zookeeper_hosts: localhost:2181  # Deprecated
```

Comment:
Are these deprecated comments to remind us to update in the future, or are we not using these settings?

Reply:
I could not determine this myself, and introduced this question in a new issue: https://github.com/pulibrary/princeton_ansible/issues/315